public/src/components/channelManagement/bannerTests/bannerTestVariantEditorCtasEditor.tsx

```diff
@@ -33,7 +33,7 @@ interface VariantEditorButtonsEditorProps extends WithStyles<typeof styles> {
   supportSecondaryCta: boolean;
 }
 
-const VariantEditorButtonsEditor: React.FC<VariantEditorButtonsEditorProps> = ({
+const BannerTestVariantEditorCtasEditor: React.FC<BannerTestVariantEditorCtasEditorProps> = ({
```

Comment:
just a name, but I wonder if this could just be `BannerCtasEditor`

Reply:
yeah the naming convention is a little clunky here. I think it might be best to do a follow up PR to clean that up a little (being thorough will touch quite a few files!)

Reply:
👍